egs/librispeech/s5/local/lm/train_lm.sh

```diff
@@ -47,7 +47,7 @@ split_prefix=$tmp_dir/split
 if [ "$stage" -le 1 ]; then
   mkdir -p $tmp_dir
   echo "Splitting into $normjobs parts, to allow for parallel processing ..."
-  split_files=$(eval "echo $split_prefix-{$(seq -s',' $normjobs)}")
+  split_files=$(for n in $(seq $normjobs); do echo $split_prefix-$n; done)
```

Comment:
I think it's better if you revert this change, that will avoid me having to verify that it's correct.